### ISSUE TITLE
RLadies Rome talk - Nov 18

### ISSUE THREAD

**Issue by jules32** [Openscapes/how_we_work#386]
10 am PT / 7 pm Roma time
Invited by Federica Gazzelloni

Talk about movement building and forking as a world view

**Comment by jules32** [comment]
**Bio**: **Julia Stewart Lowndes, PhD**
Openscapes core team member and founding director

I am a marine ecologist working at the intersection of actionable science, data science, and open science. My main focus is mentoring teams to develop technical and leadership mindsets and skills for data-intensive research, grounded in climate solutions, inclusion, and kindness. I earned my PhD from Stanford University in 2012 studying drivers and impacts of Humboldt squid in a changing climate. I am an active participant in the open science community, and co-founded RLadies Santa Barbara. In 2018 I founded Openscapes as an open source community following my own research team’s path to better science in less time, as a Mozilla Fellow and Senior Fellow at the National Center for Ecological Analysis and Synthesis (NCEAS) at the University of California Santa Barbara. In 2022 I started Openscapes LLC, a woman-owned small business to support the growing Openscapes open source community.

**Title**: **Forking as a worldview**

**Abstract:**
Forking is a concept from software development, which has evolved with open source software and the internet. Forking means taking something that works and is made available, and adapting it to your own use, while being networked back to the source – this networking serves dual purposes of being able to contribute back and also to provide appropriate credit. Forking “something” has traditionally been code, but it can be applied to nearly anything. It is about taking something that works to new places to solve problems faster. Since 2018, Openscapes has been working with teams in government and academia to help upskill science teams to work on big collaborative challenges through open science. With NASA Openscapes, this big challenge is upskilling people that use NASA Earthdata to migrate their workflows to cloud computing, as NASA Earthdata holdings migrate to the cloud. We’ve supported this effort through developing a mentor community across NASA Earth science data centers (DAACs) to co-create common tutorials and develop skills for teaching, mentoring, facilitation. And, in the process, we’ve forked a lot of code, documentation, workshop setup, slides, art, teaching approaches, and even whole programs like NOAA Fisheries Openscapes and Pathways to Open Science. We’ve come to realize that forking is a way of thinking, a worldview. Forking is an important thing to call out as we think about how we innovate together through open science, and I'm excited to talk about it with you all!
